index.js

```diff
@@ -267,4 +263,13 @@ Emailer.admin = {
     }
 };
 
+Emailer.addMiddleware = function(data, callback) {
+    data.app.head('/emailer-mandrill/reply', function(req, res) {
+        res.sendStatus(200);
+    });
+    data.app.post('/emailer-mandrill/reply', Emailer.receive);
+
+    callback();
+};
+
```

Comment:
May I ask why you added the routes here instead of in `static:app.load` like before?